Formula/bazel.rb

```diff
@@ -15,6 +16,9 @@ class Bazel < Formula
 
   def install
     ENV["EMBED_LABEL"] = "#{version}-homebrew"
+    # Bazel has it owns sandbox, ensure it works.
+    ENV.delete("SDKROOT")
+    ENV["CC"] = `xcrun -f #{ENV["CC"]}`.strip
```

Comment:
I might be wrong, but stripping things like `MACOSX_DEPLOYMENT_TARGET` or `SDKROOT` from the environment on OS X sounds like broken build system behavior to me. I can sympathize with the desire to take control over one's build environment, but please understand that a package manager also has to exercise some control over these parameters to achieve a certain degree of predictable build output. And countless hours have been spent making things work with various OS X versions, in the presence/absence of Xcode and/or the Command Line Tools, etc.

Also, `superenv` does a lot more than just find an appropriate compiler. Is there no way to sanitize the environment, but allow for a whitelist of variables that should be retained? I haven't checked thoroughly, but keeping everything with the `HOMEBREW_` prefix and the above mentioned key variables will likely work. Or an option to opt-out of the filtering and leave that job to Homebrew?